docs/tutorials/average_optimizers_callback.ipynb

```diff
@@ -76,9 +76,9 @@
         "\n",
```

Comment:
<p>Might be a good idea to add a note to mention SWA needs to have an LR schedule in order to get to the center of a loss basin.</p><p> </p><p>Usage of "we" is causing the <code>google:second person</code>  error I think. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/2497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I see, i was a bit confused on that part thanks for sharing the blog. About "we", I think I've removed every instance of it and yet it throws error....</p>

Reply:
@MarkDaoust what could be the reason?

Reply:
" We will use default params here" on line 223